datafusion/proto/proto/datafusion.proto

```diff
@@ -90,7 +90,8 @@ message CsvFormat {
 }
```

Comment:
I recommend viewing this with whitespace disabled

![image](https://user-images.githubusercontent.com/1781103/209133218-de838a6d-bea8-4681-865a-a5ee3f3299eb.png)